draft-packages/illustration/package.json

```diff
@@ -35,7 +35,7 @@
     "@kaizen/hosted-assets": "^1.1.0"
   },
   "devDependencies": {
-    "@cultureamp/elm-storybook": "cultureamp/elm-storybook#0.1.0",
+    "elm-storybook": "cultureamp/elm-storybook#0.1.0",
```

Comment:
Should this be 0.3.0?

It seems this was the only package still on 0.1.0 so may have been missed in a find/replace

Reply:
It wasn't actually used in this package (no Elm version, I'm guessing it just came from the boilerplate). I'm going to bump it for consistency.